api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/support/CasConfigurationJasyptCipherExecutor.java

```diff
@@ -225,21 +246,31 @@ private void initializeJasyptInstanceIfNecessary() {
     public enum JasyptEncryptionParameters {
 
         /**
-         * Jasypt algorithm name to use.
+         * Jasypt algorithm name to use. Default is PBEWithMD5AndTripleDES.
          */
         ALGORITHM("cas.standalone.configuration-security.alg", "PBEWithMD5AndTripleDES"),
         /**
-         * Jasypt provider name to use.
+         * Jasypt provider name to use. None for Java, BC for BouncyCastle.
          */
         PROVIDER("cas.standalone.configuration-security.provider", null),
         /**
          * Jasypt number of iterations to use.
          */
         ITERATIONS("cas.standalone.configuration-security.iterations", null),
         /**
-         * Jasypt password to use.
+         * Jasypt password to use for encryption and decryption.
+         */
+        PASSWORD("cas.standalone.configuration-security.psw", null),
+        /**
+         * Use (or not) a Jasypt Initialization Vector (IV).
+         * An initialization vector must be enabled with `PBEWithDigestAndAES` algorithms that aren't BouncyCastle.
```

Comment:
Please make sure all *code* elements, such as `PBEWithDigestAndAES `, are wrapped in `{@code xyz}`.